### PR TITLE
MAT 5 Cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/feast_of_the_victorious_dead.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Feast of the Victorious Dead
+ManaCost:W B
+Types:Enchantment
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | CheckSVar$ Y | Execute$ TrigGainLife | TriggerDescription$ At the beginning of your end step, if one or more creatures died this turn, you gain that much life and distribute that many +1/+1 counters among creatures you control.
+SVar:TrigGainLife:DB$ GainLife | Defined$ You | LifeAmount$ Y | SubAbility$ DBPutCounter
+SVar:DBPutCounter:DB$ PutCounter | ValidTgts$ Creature.YouCtrl | ChoiceTitle$ Choose any number of creatures you control to distribute counters to | CounterType$ P1P1 | CounterNum$ Y | DividedAsYouChoose$ Y | TargetMin$ 1 | TargetMax$ Y
```
Comment (squee1968) [inline forge-gui/res/cardsfolder/upcoming/feast_of_the_victorious_dead.txt:6]: Doesn't target

File: forge-gui/res/cardsfolder/upcoming/karn_legacy_reforged.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Name:Karn, Legacy Reforged
+ManaCost:5
+Types:Legendary Artifact Creature Golem
+PT:*/*
+S:Mode$ Continuous | EffectZone$ All | CharacteristicDefining$ True | IsPresent$ Card.Self+notattacking | SetPower$ X | SetToughness$ X | Description$ CARDNAME's power and toughness are each equal to the greatest mana value among artifacts you control.
```
Comment (squee1968) [inline forge-gui/res/cardsfolder/upcoming/karn_legacy_reforged.txt:5]: remove +nonattacking

File: forge-gui/res/cardsfolder/upcoming/jirina_dauntless_general.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Name:Jirina, Dauntless General
+ManaCost:W B
+Types:Legendary Creature Human Soldier
+PT:2/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigExile | TriggerDescription$ When CARDNAME enters the battlefield, exile all cards from target player's graveyard.
+SVar:TrigExile:DB$ ChangeZoneAll | ValidTgts$ Player | TgtPrompt$ Select target player | Origin$ Graveyard | Destination$ Exile | ChangeType$ Card | IsCurse$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/jirina_dauntless_general.txt:6]: TgtPrompt unneeded

File: forge-gui/res/cardsfolder/upcoming/jirina_dauntless_general.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+A:AB$ PumpAll | Cost$ Sac<1/CARDNAME> | ValidCards$ Human.YouCtrl | KW$ Hexproof & Indestructible | SpellDescription$ Humans you control gain hexproof and indestructible until end of turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/jirina_dauntless_general.txt:7]: NICKNAME

**Review by tool4ever — APPROVED** [review]

**Pull request merged**